src/Mvc/test/Mvc.FunctionalTests/ApiBehaviorTest.cs

```diff
@@ -82,6 +83,73 @@ public virtual async Task ActionsReturnBadRequest_WhenModelStateIsInvalid()
         }
     }
 
+    [Fact]
+    public virtual async Task ActionWithNonOptionalParameterReturnsNullIsNotPermited_WhenInvokedWithNullParameter()
+    {
+        // Arrange
+        using (new ActivityReplacer())
+        {
+            // Act
+            var response = await Client.PostAsJsonAsync<Contact>("/contact", default);
+
+            // Assert
+            await response.AssertStatusCodeAsync(HttpStatusCode.BadRequest);
+            Assert.Equal("application/problem+json", response.Content.Headers.ContentType.MediaType);
+            var problemDetails = JsonConvert.DeserializeObject<ValidationProblemDetails>(
+                await response.Content.ReadAsStringAsync(),
+                new JsonSerializerSettings
+                {
+                    Converters = { new ValidationProblemDetailsConverter() }
+                });
+
+            Assert.Equal("One or more validation errors occurred.", problemDetails.Title);
+            Assert.Equal("https://tools.ietf.org/html/rfc7231#section-6.5.1", problemDetails.Type);
+
+            Assert.Collection(
+                problemDetails.Errors.OrderBy(kvp => kvp.Key),
+                kvp =>
+                {
+                    var error = Assert.Single(kvp.Value);
+                    Assert.Equal("null value not supported", error);
+                }
+            );
+        }
+    }
+
+    [Fact]
+    public virtual async Task ActionWithNonOptionalParameterReturnsBodyCannotBeEmpty_WhenInvokedWithEmptyBody()
```

Comment:
```suggestion
    public Task JsonInputFormatter_ReturnsBadRequest_ForEmptyRequestBodyAction()
```

Reply:
As mentioned, maybe you should move it to https://github.com/brunolins16/aspnetcore/blob/main/src/Mvc/test/Mvc.FunctionalTests/SystemTextJsonInputFormatterTest.cs